coordinator/src/main/scala/filodb.coordinator/QueryActor.scala

```diff
@@ -87,9 +89,22 @@ final class QueryActor(memStore: TimeSeriesStore,
   private val tags = Map("dataset" -> dsRef.toString)
   private val lpRequests = Kamon.counter("queryactor-logicalPlan-requests").withTags(TagSet.from(tags))
   private val epRequests = Kamon.counter("queryactor-execplan-requests").withTags(TagSet.from(tags))
-  private val resultVectors = Kamon.histogram("queryactor-result-num-rvs").withTags(TagSet.from(tags))
```

Comment:
Is this intentional?

Reply:
yes - intentional. Not being plotted in dashboard. Not helpful with detailed debugging. Result size already tracked with other metrics.